credentials/java/com/google/auth/Credentials.java

```diff
@@ -54,6 +56,19 @@ public abstract class Credentials implements Serializable {
    */
   public abstract String getAuthenticationType();
 
+  /**
+   * Returns the universe domain for the credential.
```

Comment:
Added, thanks!

Reply:
Not sure I understand why we mention token refreshes or that is it blocking? Seems reasonable that all calls are blocking unless the API specifies otherwise.